### PR TITLE
Update tailscale name resolution on examples

### PR DESCRIPTION
**Pull request by guilhermedemouraa**
Motivation: https://github.com/farm-ng/amiga-dev-kit/issues/279

### CODE REVIEW

File: website/docs/concepts/system_overview/README.md [hunk 1]
```diff
@@ -75,3 +73,40 @@ Below is the architectural diagram that visually represents how these services i
...
+A key feature of our development environment is its flexibility; examples can be executed on your
+local machine as well as directly on the Amiga's brain.
+This adaptability is essential for developers who wish to work within their own environments or
+need to manage operations across multiple robots.
+
+#### Running Examples on Your Local Machine
+
+To run examples from your local machine and communicate with the Amiga robot over Wifi, you must
+modify the `service_config.json` file (or its equivalent).
+Specifically, the `host` field should be changed from `localhost` to your robot's unique tailscale name
```
Comment (Hackerman342) [inline website/docs/concepts/system_overview/README.md:99]: Add a comment about + link to the instructions for configuring tailscale

File: website/docs/examples/camera_aruco_detector/README.md [hunk 2]
```diff
@@ -250,7 +250,9 @@ async for event, message in camera_client.subscribe(config.subscriptions[0], dec
 ### 5. Execute the Python script
 
 To run this script from your PC, you need to update the `service_config.json`
-by modifying the `host` filed with your Amiga brain name (e.g., `element-vegetable`).
+by modifying the `host` field with your Amiga brain name.
+Please check out the [Amiga Development 101](../../concepts/system_overview/README.md#where-to-run-the-examples)
```
Comment (Hackerman342) [inline website/docs/examples/camera_aruco_detector/README.md:254]: Use doc paths for links. Please fix this here and in all other examples
```suggestion
Please check out the [Amiga Development 101](/docs/concepts/system_overview/#where-to-run-the-examples)
```

File: website/docs/concepts/system_overview/README.md [hunk 3]
```diff
@@ -75,3 +73,40 @@ Below is the architectural diagram that visually represents how these services i
...
+(e.g., `element-vegetable.tail0be07.ts.net`).
+This ensures that your gRPC client can successfully connect to the robot, facilitating seamless
+interaction and data exchange.
+
```
Comment (Hackerman342) [inline website/docs/concepts/system_overview/README.md:103]: Suggest to have an example here of changing host field, and remove it from all examples:

> Notice how we change the `host` field here from `localhost` to `element-vegetable.tail0be07.ts.net`:

```json
{
    "name": "filter",
    "port": 20001,
    "host": "element-vegetable.tail0be07.ts.net",
    "log_level": "INFO",
    "subscriptions": [
        {
            "uri": {
                "path": "/state",
                "query": "service_name=filter"
            },
            "every_n": 1
        }
    ]
}
```

File: website/docs/examples/filter_client/README.md [hunk 4]
```diff
@@ -86,28 +86,29 @@ By default, the host address is assumed to be `localhost`.
...
+    "subscriptions": [
+        {
+            "uri": {
+                "path": "/state",
+                "query": "service_name=filter"
+            },
+            "every_n": 1
+        }
+    ]
+}
```
Comment (Hackerman342) [inline website/docs/examples/filter_client/README.md:109]: Suggest to remove this from all the examples and have an example of changing the `host` in the single source of truth

* Suggestion is missing removing the final backticks, so make sure not to commit directly
```suggestion
```

**Review by Hackerman342 — APPROVED** [review]
Thanks @guilhermedemouraa !

File: website/docs/concepts/system_overview/README.md (hunk 1 above)
Comment (guilhermedemouraa) [reply website/docs/concepts/system_overview/README.md]: Good suggestion. Implemented here: 0e04b7a884259bc3c9baa37585235f275e6abd57

File: website/docs/concepts/system_overview/README.md (hunk 3 above)
Comment (guilhermedemouraa) [reply website/docs/concepts/system_overview/README.md:111]: 0e04b7a884259bc3c9baa37585235f275e6abd57

File: website/docs/examples/camera_aruco_detector/README.md (hunk 2 above)
Comment (guilhermedemouraa) [reply website/docs/examples/camera_aruco_detector/README.md]: Done here: b5cef04c9604e21e0de401d3c546fbe0382efd1f

**Pull request merged**